asdf/tags/core/external_reference.py

```diff
@@ -0,0 +1,64 @@
+from ...asdftypes import AsdfType
+
+
+class ExternalArrayReference(AsdfType):
+    """
+    Store a reference to an array in an external File.
+
+    This class is a simple way of referring to an array in another file. It
+    provides no way to resolve these references, that is left to the user.
```

Comment:
Might want to add here that it doesn't even check whether the externally referenced file exists or not.